snuba/cli/query_replayer.py

```diff
@@ -8,20 +9,25 @@
 
 from snuba import settings
 from snuba.admin.notifications.slack.client import SlackClient
+from snuba.clickhouse.errors import ClickhouseError
 from snuba.clickhouse.native import ClickhousePool
 from snuba.clickhouse.upgrades.comparisons import (
     BlobGetter,
     FileFormat,
     FileManager,
     QueryInfoResult,
     QueryMeasurementResult,
+    delete_local_file,
+    full_path,
 )
 from snuba.clusters.cluster import ClickhouseClientSettings
 from snuba.environment import setup_logging, setup_sentry
 from snuba.utils.gcs import GCSUploader
 
 logger = structlog.get_logger().bind(module=__name__)
 
+TABLE_DOESNT_EXIST = 60
```

Comment:
If this is a clickhouse error, i believe they should expose it. If yes, lets use that instead.

Reply:
ah yeah clickhouse driver has them in the `ErrorCodes`